backend/python/requirements.txt

```diff
@@ -16,8 +17,6 @@ google-api-core==1.26.3
 google-api-python-client==2.2.0
 google-auth==1.29.0
 google-auth-httplib2==0.1.0
-googleapis-common-protos==1.53.0
-httplib2==0.19.1
```

Comment:
Were these automatically uninstalled? :o

Reply:
No they were just being repeated in the file for some reason.